sgl-kernel/src/sgl-kernel/ops/__init__.py

```diff
@@ -1,5 +1,20 @@
+from .custom_reduce_cuda import all_reduce as _all_reduce
+from .custom_reduce_cuda import dispose as _dispose
+from .custom_reduce_cuda import init_custom_ar as _init_custom_ar
 from .warp_reduce_cuda import reduce as _reduce
 
 
 def warp_reduce(input_tensor):
     return _reduce(input_tensor)
+
+
+def init_custom_reduce(rank_id, num_devices, buffers, barrier_in, barrier_out):
```

Comment:
Please add unit tests for these, comparing them with vLLM's implementation.

Reply:
https://github.com/sgl-project/sglang/tree/main/sgl-kernel/tests
Tests can be added here, and we could also integrate them into CI.

Reply:
there is correctness test and performance test with vllm

Reply:
cool I'll help verify locally